src/cursorless.talon

```diff
@@ -1,22 +1,21 @@
 app: vscode
 -
-tag(): user.cursorless
```

Comment:
wait so you've removed the tag altogether?  I thought we decided that the tag was nicer than directly tying everything to vscode

Reply:
The voice commands are only activated by vscode. The internal tag was only for the lists. See previous comment on that.